requests.go

```diff
@@ -19,14 +19,14 @@ import (
 )
 
 const (
-	LDContextDisclosureRequest = "https://irma.app/ld/request/disclosure/v2"
-	LDContextSignatureRequest  = "https://irma.app/ld/request/signature/v2"
-	LDContextIssuanceRequest   = "https://irma.app/ld/request/issuance/v2"
-	LDContextRevocationRequest = "https://irma.app/ld/request/revocation/v1"
-	LDContextOptionsRequest    = "https://irma.app/ld/request/options/v1"
-	LDContextClientRequest     = "https://irma.app/ld/request/client/v1"
-	LDContextSessionOptions    = "https://irma.app/ld/options/v1"
-	DefaultJwtValidity         = 120
+	LDContextDisclosureRequest      = "https://irma.app/ld/request/disclosure/v2"
+	LDContextSignatureRequest       = "https://irma.app/ld/request/signature/v2"
+	LDContextIssuanceRequest        = "https://irma.app/ld/request/issuance/v2"
+	LDContextRevocationRequest      = "https://irma.app/ld/request/revocation/v1"
+	LDContextFrontendOptionsRequest = "https://irma.app/ld/request/frontendoptions/v1"
+	LDContextClientRequest          = "https://irma.app/ld/request/client/v1"
+	LDContextSessionOptions         = "https://irma.app/ld/options/v1"
+	DefaultJwtValidity              = 120
 )
```

Comment:
No, I was planning to rename `irma.Qr` -> `irma.SessionPointer` in a later PR, because that is a rather large change not related to this PR. `irma.ClientRequest` -> `irma.ClientSessionRequest` I forgot, to be honest, but is related to this PR, and a smaller change, so I better include it here.